Specs/Core/loadKTX2Spec.js

```diff
@@ -0,0 +1,194 @@
+import { loadKTX2 } from "../../Source/Cesium.js";
+import { KTX2Transcoder } from "../../Source/Cesium.js";
+import { PixelFormat } from "../../Source/Cesium.js";
+import { Resource } from "../../Source/Cesium.js";
+import { RuntimeError } from "../../Source/Cesium.js";
+
+describe("Core/loadKTX2", function () {
```

Comment:
@lilleyse These should be good for review now. I cleaned up some tests and removed a few others:
- Removed "mipmaps are supported" test because `OctahedralProjectedCubeMapSpec` confirms this
- Removed a couple tests checking for errors when the .ktx2 had mismatching type/typeSize with compressed textures. I could add these back, but since compressed ktx2 buffers don't always fit evenly into Uint32Arrays it's require more precise byte changes to manipulate the header info. 

Do we want to throw specific errors when header magic fields like `vkFormat` etc. are invalid? I expect these to be handled as "Invalid KTX2 file." errors through the transcoder module.

Reply:
The catch-all "Invalid KTX2 file" is ok for now. If it becomes a problem we can make the errors more granular.